api/db/migrations/003_add_calendly_resource.up.sql

```diff
@@ -0,0 +1,14 @@
+/* calendlys table */
+CREATE TABLE IF NOT EXISTS calendlys (
+    id UUID NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    link TEXT NOT NULL,
+    interviewee TEXT REFERENCES users ON DELETE RESTRICT,
```

Comment:
I think we want the following to support the one (interviewer) to many (interviewee) relationship
```suggestion
    interviewee TEXT[] REFERENCES users ON DELETE RESTRICT,
```

Reply:
I've added this change @armiantos . Ready for rereview.